tests/unit/test_context.py

```diff
@@ -342,3 +343,30 @@ def test_filepath(tmpdir):
     assert c.schema["root"].tables["df"].filepath is None
     with pytest.raises(KeyError):
         c.schema["root"].filepaths["df"]
+
+
+def test_ddf_filepath(tmpdir):
```

Comment:
Does the parquet_ddf fixture work for the test here, or does it explicitly need to be redefined here?